addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -759,6 +759,47 @@ def test_in_invoice_line_onchange_taxes_1(self):
             'amount_total': 1384.0,
         })
 
+    def test_in_invoice_change_date_rounding_multi_currency(self):
+        new_currency_id = self.env['res.currency'].create({
+                'name': 'Test',
+                'symbol': 'T',
+                'rounding': 0.01,
+            })
+
+        self.env['res.currency.rate'].create([
+            {
+                'currency_id': new_currency_id.id,
+                'name': '2017-01-01',
+                'rate': 0.233661237937,
+            }, {
+                'currency_id': new_currency_id.id,
+                'name': '2016-01-01',
+                'rate': 0.233666697822,
+            }
+        ])
+
+        tax_20 = self.env['account.tax'].create({
+            'name': "tax",
+            'amount_type': 'percent',
+            'amount': 20.0
+        })
+
+        move_form = Form(self.env['account.move'].with_context(default_move_type='in_invoice', check_move_validity=True))
```

Comment:
Do you need to use a form?

Reply:
The form is necessary since the error happens on `save` which can't be tested on a raw `account.move` record. I have removed the tax line though for simplicity. Let me know if you have any additional thoughts!